dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/testing/ContributionIntegrationTest.java

```diff
@@ -88,29 +88,29 @@ public void setupHelper(){
     @Test
     void testProcessContributionUpdateWhenNotFound() {
         final String errorText = "The request has failed to process";
-        final var updateLogContributionRequest = UpdateLogContributionRequest.builder()
+        final var updateLogContributionRequest = ContributionProcessedRequest.builder()
```

Comment:
This local variable name seems to be a hangover from the previous name and can be changed to contributionProcessedRequest?